packages/react/src/Bento/Bento.module.css

```diff
@@ -0,0 +1,373 @@
+  grid-column: auto/span 10;
+}
+
+.Bento__Item--column-span-11 {
+  grid-column: auto/span 11;
+}
+
+.Bento__Item--column-span-12 {
+  grid-column: auto/span 12;
+}
+
+/* Row Span */
+.Bento__Item--row-span-1 {
+  grid-row: auto/span 1;
+}
+
+.Bento__Item--row-span-2 {
+  grid-row: auto/span 2;
+}
+
+.Bento__Item--row-span-3 {
+  grid-row: auto/span 3;
+}
+
+.Bento__Item--row-span-4 {
+  grid-row: auto/span 4;
+}
+
+.Bento__Item--row-span-5 {
+  grid-row: auto/span 5;
+}
+
+.Bento__Item--row-span-6 {
+  grid-row: auto/span 6;
+}
+
+.Bento__Item--row-span-7 {
+  grid-row: auto/span 7;
+}
+
+.Bento__Item--row-span-8 {
+  grid-row: auto/span 8;
+}
+
+.Bento__Item--row-span-9 {
+  grid-row: auto/span 9;
+}
+
+.Bento__Item--row-span-10 {
+  grid-row: auto/span 10;
+}
+
+.Bento__Item--row-span-11 {
+  grid-row: auto/span 11;
+}
+
+.Bento__Item--row-span-12 {
+  grid-row: auto/span 12;
+}
+
+/* Column Start */
+.Bento__Item--column-start-1 {
+  grid-column-start: 1;
+}
+
+.Bento__Item--column-start-2 {
+  grid-column-start: 2;
+}
+
+.Bento__Item--column-start-3 {
+  grid-column-start: 3;
+}
+
+.Bento__Item--column-start-4 {
+  grid-column-start: 4;
+}
+
+.Bento__Item--column-start-5 {
+  grid-column-start: 5;
+}
+
+.Bento__Item--column-start-6 {
+  grid-column-start: 6;
+}
+
+.Bento__Item--column-start-7 {
+  grid-column-start: 7;
+}
+
+.Bento__Item--column-start-8 {
+  grid-column-start: 8;
+}
+
+.Bento__Item--column-start-9 {
+  grid-column-start: 9;
+}
+
+.Bento__Item--column-start-10 {
+  grid-column-start: 10;
+}
+
+.Bento__Item--column-start-11 {
+  grid-column-start: 11;
+}
+
+.Bento__Item--column-start-12 {
+  grid-column-start: 12;
+}
+
+/* Row Start */
+.Bento__Item--row-start-1 {
+  grid-row-start: 1;
+}
+
+.Bento__Item--row-start-2 {
+  grid-row-start: 2;
+}
+
+.Bento__Item--row-start-3 {
+  grid-row-start: 3;
+}
+
+.Bento__Item--row-start-4 {
+  grid-row-start: 4;
+}
+
+.Bento__Item--row-start-5 {
+  grid-row-start: 5;
+}
+
+.Bento__Item--row-start-6 {
+  grid-row-start: 6;
+}
+
+.Bento__Item--row-start-7 {
+  grid-row-start: 7;
+}
+
+.Bento__Item--row-start-8 {
+  grid-row-start: 8;
+}
+
+.Bento__Item--row-start-9 {
+  grid-row-start: 9;
+}
+
+.Bento__Item--row-start-10 {
+  grid-row-start: 10;
+}
+
+.Bento__Item--row-start-11 {
+  grid-row-start: 11;
+}
+
+.Bento__Item--row-start-12 {
+  grid-row-start: 12;
+}
+
+/* Bento Item Vertical Align */
+.Bento__Item-verticalAlign--start {
+  align-items: start;
+}
+
+.Bento__Item-verticalAlign--center {
+  align-items: center;
+}
+
+.Bento__Item-verticalAlign--end {
+  align-items: end;
+}
+
+/* Bento Item Horizontal Align */
+.Bento__Item-horizontalAlign--top {
+  justify-content: start;
+}
+
+.Bento__Item-horizontalAlign--center {
+  justify-content: center;
+}
+
+.Bento__Item-horizontalAlign--end {
+  justify-content: end;
+}
+
+/* Bento Content */
+.Bento__Item--visual-as-background .Bento__Content {
+  order: 1;
+  grid-row: 1 / span 2;
+  grid-column: 1 / span 2;
+}
+
+.Bento__Content {
```

Comment:
The content area needs an inner padding of `56px`

Reply:
A static 56px rather than a configurable padding value like for Visual. The 56px would look like this on mobile:
![image](https://github.com/primer/brand/assets/31220082/461f1d5e-c196-445d-9ce0-985b22574360)

Reply:
I think we'll need something configurable, like the border radius or the gap between items in bento

Reply:
So I guess my question is if we should provide a map as I have provided to specific values or should we not allow updating or some mix of both? Let's chat Monday about this. I would love to get your input.